packages/react-server-test-pages/pages/bottleneck/dataRequests.js

```diff
@@ -0,0 +1,12 @@
+import {ReactServerAgent} from "react-server";
+
+export default class DataRequestsPage {
+
+	getElements() {
+		//Perform 1 million data requests before returning
+		for (var i = 0; i < 999999; i++) {
+			ReactServerAgent.get('/data/delay');
+		}
+		return (<div>Data requests complete!</div>);
+	}
+}
```

Comment:
Technically I think we'd want the data requests to be kicked off in `handleRoute`. Practically, I'm not sure if that will make a difference.